chromium_src/chrome/browser/process_singleton_posix.cc

```diff
@@ -348,6 +348,21 @@ bool CheckCookie(const base::FilePath& path, const base::FilePath& cookie) {
   return (cookie == ReadLink(path));
 }
 
+bool IsAppSandboxed() {
+#if defined(OS_MACOSX)
+  // NB: There is no sane API for this, we have to just guess by
```

Comment:
That's the whole problem, if you're sandboxed but _not_ using the MAS build (i.e. because you just want to sandbox your app), this ifdef is incorrect